plugin/pkg/auth/authorizer/rbac/rbac.go

```diff
@@ -62,7 +62,7 @@ func (r *RBACAuthorizer) Authorize(attr authorizer.Attributes) error {
 		}
 	}
 
-	return validation.ConfirmNoEscalation(ctx, r.authorizationRuleResolver, []rbac.PolicyRule{requestedRule})
+	return validation.ConfirmNoEscalation(ctx, r.authorizationRuleResolver, []rbac.PolicyRule{requestedRule}), nil
```

Comment:
This seemed like the correct way to answer the question "does this context have access to this resource" (just pass a single rule rather than multiple). Don't we have to ask that question in some capacity, even if it's not through the rule resolver?